src/main.rs

```diff
@@ -1,18 +1,22 @@
+use std::fs::File;
+use std::io::Read;
 use std::sync::mpsc;
 use std::time::Duration;
 use std::{cmp::Ordering, collections::HashSet};
 
+use anyhow::{ format_err, Result };
```

Comment:
Remove the whitespaces around the brackets:

```suggestion
use anyhow::{format_err, Result};
```